test/functional/cmdLineTests/loadLibraryTest/playlist.xml

```diff
@@ -40,4 +40,4 @@
 			<group>functional</group>
 		</groups>
 	</test>
-</playlist>
\ No newline at end of file
+</playlist>
```

Comment:
Could we remove this change as there is only a newline added in the file?

Reply:
removed